sfcli/SmartFace.Cli/Infrastructure/ApiImplementation/CamerasRepository.cs

```diff
@@ -77,46 +75,12 @@ private Camera UpdateCameraData(Camera originalCamera, CameraRequestData updateD
             updatedCamera.FaceDetectorConfig.MaxFaceSize = updateData.TrackMaxFaceSize ?? updatedCamera.FaceDetectorConfig.MaxFaceSize;
             updatedCamera.FaceDetectorConfig.MinFaceSize = updateData.TrackMinFaceSize ?? updatedCamera.FaceDetectorConfig.MinFaceSize;
 
-            //TODO when the API takes the resources as string, maybe just forward them, no mapping needed
-            updatedCamera.TemplateGeneratorResourceId = MapTemplateResourceId(updateData.TemplateGeneratorResourceId) ?? updatedCamera.TemplateGeneratorResourceId;
-            updatedCamera.FaceDetectorResourceId = MapFaceDetectorResourceId(updateData.FaceDetectorResourceId) ?? updatedCamera.FaceDetectorResourceId;
+            updatedCamera.TemplateGeneratorResourceId = updateData.TemplateGeneratorResourceId;
```

Comment:
I think we should follow the same pattern with `updatedCamera.TemplateGeneratorResourceId = updateData.TemplateGeneratorResourceId ?? updatedCamera.TemplateGeneratorResourceId;`, so that we don't overwrite the resource ID when updating other fields

Reply:
done